src/transaction/transaction_group_complete_manager.cpp

```diff
@@ -112,6 +278,10 @@ namespace cubtx
 	/* Close the current group. */
 	m_latest_closed_group_id.store (m_current_group_id);
 	m_latest_closed_group_state = GROUP_CLOSED;
+	er_log_debug (ARG_FILE_LINE,
```

Comment:
er_log_debug while mutex is acquired : might affect performance

Reply:
Same with the previous comment with the note that a parameter will be used for GC.